octgnFX/Octgn/DeckBuilder/DeckBuilderWindow.xaml.cs

```
@@ -511,6 +512,19 @@ private void CloseClicked(object sender, RoutedEventArgs e)
             Close();
         }
 
+        public bool TryClose()
+        {
+            try
+            {
+                Close();
+                return isRealClosing;
+            }
+            catch
+            {
```

In the future, if you wouldn't make sure and `Log.Warn` instead of doing an empty catch block.